web/modules/custom/demo_content/content/rdf_entity/asset_distribution/blue.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
This is more practical, sorry. You don't need to code the content. If you need a new node or rdf entity., you just create it manually and then with `drush dce node {node}` you export it in JSON. No manual coding. This is the accepted standard way to provide default content in D8.